homeassistant/components/home_connect/const.py

```diff
@@ -95,14 +95,19 @@
 SERVICE_SETTING = "change_setting"
 SERVICE_START_PROGRAM = "start_program"
 
+ATTR_ALLOWED_VALUES = "allowedvalues"
 ATTR_AMBIENT = "ambient"
 ATTR_BSH_KEY = "bsh_key"
+ATTR_CONSTRAINTS = "constraints"
 ATTR_DESC = "desc"
 ATTR_DEVICE = "device"
 ATTR_KEY = "key"
+ATTR_MAX = "max"
+ATTR_MIN = "min"
```

Comment:
Aren't these in `homeassistant.const`?

Reply:
If we use those values, would it be okay if I use `"max"` and `"min"` in the test to ensure that if the constants' values change in the future, we'll know that we need other constants?